### PR TITLE
Add unit tests for like.js

### PR DESCRIPTION
**Pull request by marvingay**
https://github.com/TheOdinProject/theodinproject/issues/1506

This commit:
- Adds unit tests for the like.js component
- Adds aria label to icon in component for accessibility purposes.

### CODE REVIEW

File: app/javascript/components/project-submissions/components/__tests__/like.test.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+import React from 'react';
+import ProjectSubmissionContext from '../../ProjectSubmissionContext';
+import Like from '../like';
+
+describe('Like', () => {
+  const renderLikeComponent = (userId = null, isLikedByUser = false, handleClick = () => {}) => {
+    return render(
+    )
+  }
+
```
Comment (leosoaivan) [inline app/javascript/components/project-submissions/components/__tests__/like.test.js:11]: Remove?

File: app/javascript/components/project-submissions/components/__tests__/like.test.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        />;
+      </ProjectSubmissionContext.Provider>
+    );
+    const anchorNode = getByLabelText(/like/);
+    fireEvent.click(anchorNode);
+
+    expect(handleClick).toHaveBeenCalledTimes(1);
+  })
+
+})
```
Comment (leosoaivan) [inline app/javascript/components/project-submissions/components/__tests__/like.test.js:80]: Add newline.

File: app/javascript/components/project-submissions/components/like.js [hunk 3]
```diff
@@ -14,7 +14,7 @@ const Like = ({ submission, handleLikeToggle }) => {
         handleLikeToggle(submission, isCurrentUsersSubmission);
       }
       }>
-        {submission.likes} <i className={submission.is_liked_by_current_user ? 'fa fa-heart liked' : 'fa fa-heart'}></i>
+        {submission.likes} <i className={submission.is_liked_by_current_user ? 'fa fa-heart liked' : 'fa fa-heart'} aria-label='Like icon'></i>
```
Comment (leosoaivan) [inline app/javascript/components/project-submissions/components/like.js:17]: 👌

**Review by leosoaivan — CHANGES_REQUESTED** [review]

**Review by leosoaivan — APPROVED** [review]

**Comment by KevinMulhern** [comment]
🚀

**Pull request merged**